src/HostManager.cpp

```diff
@@ -214,8 +214,8 @@ void HostManager::postInterHostEvent( const Host * pHost, const TEvent & event )
     allValidHosts = afterSendingHost;
     allValidHosts.append( beforeSendingHost );
 
-    for( int j = 0, total = allValidHosts.size(); j < total; ++j ) {
-        hostList.at( allValidHosts.at( j ) )->raiseEvent( event );
+    for(int allValidHost : allValidHosts) {
```

Comment:
This seems to have the same effect unless I'm getting confused with some off-by-one increment here. I've tested it and ``raiseGlobalEvent()`` still works as expected - profile that sends out the event doesn't get it.